update macOS instructions

scrnjakovic opened the pull request:
macOS users should use `/usr/local/bin` instead of `/usr/bin` because of **SIP** or "System Integrity Protection" introduced in El Capitan update, where even root users are not allowed to edit anything in `/usr` `/bin` `/sbin` `/System` folders with exception being mentioned `/usr/local/bin`

crnjakovic committed:
update macOS instructions

macOS users should use `/usr/local/bin` instead of `/usr/bin` because of **SIP** or "System Integrity Protection" introduced in El Capitan update, where even root users are not allowed to edit anything in `/usr` `/bin` `/sbin` `/System` folders with exception being mentioned `/usr/local/bin`

niden commented:
@scrnjakovic Do you mind adding the above in the documentation right below the area you made the change? I fear that some might not use the El Capitan update and will get confused.

Other than that it looks great!

scrnjakovic commented:
Yeah, will do when I get back.

On Aug 21, 2018, at 16:02, Nikolaos Dimopoulos <notifications@github.com<mailto:notifications@github.com>> wrote:


@scrnjakovic<https://github.com/scrnjakovic> Do you mind adding the above in the documentation right below the area you made the change? I fear that some might not use the El Capitan update and will get confused.

Other than that it looks great!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/phalcon/docs/pull/1715#issuecomment-414684778>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ADuHQLSH_7BpJBwEyfL67lOPwWIvAmOLks5uTBMAgaJpZM4WD4r0>.

crnjakovic committed:
Update macOS instructions

Update instructions to be clear on which commands to use for different macOS versions
